dbms/tests/integration/test_dictionaries/configs/dictionaries/dictionary_preset_longload.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<dictionaries>
+  <dictionary>
+    <name>longload</name>
+    <source>
+      <executable>
+        <command>sleep 100 &amp;&amp; echo '5\t6';</command>
```

Comment:
For example, this dictionary blocked all dictionaries for 100 seconds. It was enough to put one such dictionary to config to hang up all dictionaries.